crates/swc_ecma_transforms_compat/src/es2021/logical_assignments.rs

```diff
@@ -33,6 +33,12 @@ impl VisitMut for Operators {
 
         match e {
             Expr::Assign(AssignExpr {
+                span,
+                op: op @ op!("&&="),
```

Comment:
`op @op!("&&=) | op @op!("||=") | op @op!("??=")` would be better

Reply:
Oh you are right. Thank you!